src/services/goToDefinition.ts

```diff
@@ -149,6 +149,25 @@ namespace ts.GoToDefinition {
         return getDefinitionFromSymbol(typeChecker, type.symbol, node);
     }
 
+    export function getDefinitionAndBoundSpan(program: Program, sourceFile: SourceFile, position: number): DefinitionInfoAndBoundSpan {
+        const definitions = getDefinitionAtPosition(program, sourceFile, position);
+
+        if (!definitions || definitions.length === 0) {
+            return undefined;
+        }
+
+        // TODO: Add textSpan for triple slash references (file and type).
+        const comment = findReferenceInPosition(sourceFile.referencedFiles, position);
+        if (comment && tryResolveScriptReference(program, sourceFile, comment) || findReferenceInPosition(sourceFile.typeReferenceDirectives, position)) {

```

Comment:
do not think you need to call tryResolveScriptReference here. just make sure definitions is not empty, which you already checked earlier.

Reply:
Fixed.